src/js/controllers/chatController.js

```diff
@@ -147,56 +143,33 @@ module.exports = ViewController.extend({
     },
 
     _receiveMessage: function(message) {
-        if (!!this.conversation) {
-
-            // we actually need to extract the appMakers first
-            // since the message rendering is done on message add event
-            // and some UI stuff is relying on the appMakers collection
-            if (!this.conversation.get('appMakers').get(message.authorId)) {
-                this.conversation.get('appMakers').add({
-                    id: message.authorId
-                });
-            }
-
-            this.conversation.get('messages').add(message);
-        }
+        return Promise.resolve(this.model.get('conversation').get('messages').add(message));
```

Comment:
Do `{merge: true}` here? Just in case you somehow get a faye message that you already fetched from the REST API?